examples/neural_sde_vae.ipynb

```diff
@@ -0,0 +1,551 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Neural SDE (VAE)\n",
+    "\n",
+    "This implementation is based on the Pytorch version of Latent SDE from [`torchsde`](https://github.com/google-research/torchsde/blob/master/examples/latent_sde.py) library. \n",
```

Comment:
Nit: put this at the bottom of this introduction; the first line is the most important and this isn't the most important piece of information.

Honestly, the theory of latent SDEs is pretty nontrivial, and the remainder of this section pretty impenetrable, so I'd start off with a sentence here that just says something very simple to the effect of "this is a VAE".

If you want to give folks a readable reference for this topic then I'd recommend also adding a link to the appropriate section of [On Neural Differential Equations](https://arxiv.org/abs/2202.02435). (I'm biased I suppose, but I definitely didn't find the original paper that clear on this front.)